/gps_slam_converter:
  ros__parameters:
    virtual_map_x: 260
    virtual_map_y: 65
    virtual_map_width: 502
    virtual_map_height: 317
    virtual_map_distance_per_pixel: 0.05
    virtual_map_point_lon: 128.8586
    virtual_map_point_lat: 35.15797
    virtual_lt_point_lon: 128.858009083
    virtual_lt_point_lat: 35.157430158
    virtual_rt_point_lon: 128.858870603
    virtual_rt_point_lat: 35.158056682

    slam_map_width: 502
    slam_map_height: 317
    intersection_start_point_lon: 128.858009083
    intersection_start_point_lat: 35.157430158
    intersection_end_point_lon: 128.858870603
    intersection_end_point_lat: 35.158056682
    lb_point_lon: 128.8584746703773
    lb_point_lat: 35.15787023134168
    rt_point_lon: 128.85864084895368
    rt_point_lat: 35.15800009142947